app.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable  */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  static clearField() {
+    const title = document.getElementById("title");
+    const author = document.getElementById("author");
+
+    title.value = "";
+    author.value = "";
+  }
+
+  static deleteBook(target) {
+    if (target.classList.contains("delete")) {
+      target.parentNode.remove();
+    }
+  }
+
+  static displayBooks() {
+    defaultBooks.forEach((book) => Book.addBooksToList(book));
+  }
+
+  static addBooksToList(book) {
+    const list = document.getElementById("form-list");
+    const row = document.createElement("tr");
+
+    row.innerHTML = `
+     ${book.title} <br>
+    ${book.author}<br>
+    <button class="delete">Remove</button>
+    `;
+
+    list.appendChild(row);
+  }
+
+  static addBook(book) {
+    const books = Book.getBooks();
+    books.push(book);
+    localStorage.setItem("books", JSON.stringify(books));
+  }
+
+  static getBooks() {
+    const books = localStorage.getItem("books");
+    if (!books) return [];
+    return JSON.parse(books);
+  }
+
+  static removeBook(title) {
+    const books = Book.getBooks();
+    const otherBook = books.filter((item) => item.title !== title);
+
+    localStorage.setItem("books", JSON.stringify(otherBook));
+  }
+}
+
+document.addEventListener("DOMContentLoaded", Book.displayBooks);
```

Comment:
- [ ] On page load, your app does not load the books from the `localStorage`. When I add books they are stored in the `localStorage` but when I reload the page, the books are not displayed. Kindly fix that.

  Books are added | Page is reloaded
  :---:|:---:
  ![matchoudi-1](https://user-images.githubusercontent.com/36542986/158177395-1af05c3a-99e8-4957-a246-e4c591b1cdc7.jpg) | ![matchoudi-2](https://user-images.githubusercontent.com/36542986/158177401-b09b04b8-d1a7-4626-9204-7ce6e773fe5b.jpg)